Replace commas with "&" in fido.search() in docs

FinityEx opened the pull request:
<!--
We know that working on code and submitting pull requests takes effort, and we appreciate your time.
Thank you.

Please be aware that everyone has to follow our code of conduct:
https://sunpy.org/coc

Furthermore, you might need to check with your work place if you are allowed to contribute code:
https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html

Also these comments are hidden when you submit this github pull request.

We have several automatic features that run when a pull request is open.
They can appear daunting but do not worry about them!
Here is a brief explanation of them:
https://docs.sunpy.org/en/latest/dev_guide/contents/pr_review_procedure.html#continuous-integration
-->

### Description

Replaced commas with ampersand in every file containig fido.search() in documentaion. From fido.search(arg1, arg2, arg3) to fido.search(arg1 & arg2 & arg3) as per request.

<!--
Provide a general description of what your pull request does.

If the pull request closes any open issues you can add this.
If you replace <Issue Number> with a number GitHub will automatically link it.
If it doesn't, please remove the following line.
-->

Fixes #5226

FinityEx commented:
> Are there any other places where we might need to make this change?

Not within docs folder, no.

nabobalis commented:
What about outside the docs folder?

FinityEx commented:
Apart from actual code files, there's no fido.search() anywhere else

nabobalis commented:
So there other examples we can change in the code files?

FinityEx commented:
Plenty of them, actually. Most of them in sunpy/sunpy/net and sunpy/examples folders, some test files etc. 
Should those be changed as well?

nabobalis commented:
I don't see why not, at least in any examples. It's not worth changing the tests.

dstansby commented on changelog/5227.trivial.rst:
```diff
@@ -0,0 +1,2 @@
+Replaced ',' with '&' in fido.search() in following doc files: docs/code_ref/net.rst, docs/guide/acquiring_data/fido.rst, docs/guide/acquiring_data/hek.rst, docs/guide/acquiring_data/jsoc.rst, docs/guide/tour.rst, docs/whatsnew/0.8.rst, docs/whatsnew/0.9.rst, docs/whatsnew/2.0.rst, 
```
I don't think there needs to be a list of filenames here, just the first bit of the changelog entry is fine.

pep8speaks commented:
Hello @FinityEx! Thanks for updating this PR. 









* In the file [`examples/acquiring_data/goes_xrs_example.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/examples/acquiring_data/goes_xrs_example.py):

> [Line 103:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/examples/acquiring_data/goes_xrs_example.py#L103): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (137 > 100 characters)
> [Line 21:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/examples/acquiring_data/goes_xrs_example.py#L21): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (112 > 100 characters)













* In the file [`examples/time_series/goes_hek_m25.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/examples/time_series/goes_hek_m25.py):

> [Line 20:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/examples/time_series/goes_hek_m25.py#L20): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)











* In the file [`sunpy/database/tables.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/database/tables.py):

> [Line 640:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/database/tables.py#L640): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/eve.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/eve.py):

> [Line 47:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/eve.py#L47): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (143 > 100 characters)
> [Line 43:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/eve.py#L43): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (154 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/fermi_gbm.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py):

> [Line 62:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py#L62): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (115 > 100 characters)
> [Line 56:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py#L56): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (126 > 100 characters)
> [Line 55:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py#L55): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)
> [Line 50:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py#L50): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)
> [Line 49:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/fermi_gbm.py#L49): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (123 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/goes.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py):

> [Line 301:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L301): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (107 > 100 characters)
> [Line 299:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L299): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (121 > 100 characters)
> [Line 298:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L298): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 100 characters)
> [Line 297:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L297): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)
> [Line 225:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L225): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)
> [Line 216:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L216): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)
> [Line 215:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L215): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (123 > 100 characters)
> [Line 214:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L214): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (116 > 100 characters)
> [Line 210:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L210): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)
> [Line 193:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L193): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (111 > 100 characters)
> [Line 192:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L192): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 100 characters)
> [Line 163:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L163): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)
> [Line 160:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L160): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 100 characters)
> [Line 145:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L145): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (117 > 100 characters)
> [Line 141:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L141): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)
> [Line 134:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L134): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (131 > 100 characters)
> [Line 78:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L78): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (106 > 100 characters)
> [Line 76:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L76): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (109 > 100 characters)
> [Line 75:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L75): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (116 > 100 characters)
> [Line 72:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L72): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (108 > 100 characters)
> [Line 70:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L70): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (124 > 100 characters)
> [Line 64:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/goes.py#L64): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (117 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/gong.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/gong.py):

> [Line 42:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/gong.py#L42): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/lyra.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/lyra.py):

> [Line 50:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/lyra.py#L50): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (111 > 100 characters)
> [Line 44:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/lyra.py#L44): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (109 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/noaa.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/noaa.py):

> [Line 151:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/noaa.py#L151): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (115 > 100 characters)
> [Line 114:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/noaa.py#L114): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)
> [Line 101:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/noaa.py#L101): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)
> [Line 46:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/noaa.py#L46): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (108 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/norh.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/norh.py):

> [Line 82:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/norh.py#L82): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)
> [Line 81:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/norh.py#L81): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)

* In the file [`sunpy/net/dataretriever/sources/rhessi.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py):

> [Line 171:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py#L171): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (101 > 100 characters)
> [Line 140:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py#L140): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (105 > 100 characters)
> [Line 136:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py#L136): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)
> [Line 85:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py#L85): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (113 > 100 characters)
> [Line 50:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/dataretriever/sources/rhessi.py#L50): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (105 > 100 characters)

* In the file [`sunpy/net/hek/hek.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/hek/hek.py):

> [Line 96:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/hek/hek.py#L96): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (103 > 100 characters)

* In the file [`sunpy/net/helio/hec.py`](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/helio/hec.py):

> [Line 153:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/helio/hec.py#L153): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (110 > 100 characters)
> [Line 148:101](https://github.com/sunpy/sunpy/blob/02b00baa430f0d8422c993f45ccd28c1dbc4a4dc/sunpy/net/helio/hec.py#L148): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (140 > 100 characters)

FinityEx commented:
Changed all files except tests, changed changelog as well. I don't know why my newest commit comment is cut in half, though. Is it because of double quotes use?

nabobalis commented:
Hi @FinityEx, sorry for the late response.

Do you mind doing two changes to this PR?

1. You will need to rebase or merge in master due to the file conflicts
2. We want to line break before the `&` if a line is too long instead of line breaking after.
